src/libs/runtime-utils.js

```diff
@@ -32,13 +32,19 @@ export const defaultNumDataprocPreemptibleWorkers = 0
 export const defaultGpuType = 'nvidia-tesla-t4'
 export const defaultNumGpus = 1
 
+export const defaultLocation = 'US'
+export const defaultLocationType = 'multi-region'
```

Comment:
I don't see that defaultLocationType and defaultComputeZone are used. Are they here for documentation purposes only, or were they intended to be used?

Reply:
Actually, I see now that defaultComputeZone is used by ComputeModal, which was collapsed in the diff.